src/Microsoft.ML.Data/Transforms/KeyToValueTransform.cs

```diff
@@ -73,6 +73,19 @@ private static VersionInfo GetVersionInfo()
         private readonly ColumnType[] _types;
         private KeyToValueMap[] _kvMaps;
 
+        /// <summary>
+        /// Convenience constructor for public facing API.
+        /// </summary>
+        /// <param name="env">Host Environment.</param>
+        /// <param name="input">Input <see cref="IDataView"/>. This is the output from previous transform or loader.</param>
+        /// <param name="name">Name of the output column.</param>
+        /// <param name="source">Name of the input column.  If this is null '<paramref name="name"/>' will be used.</param>
+        public KeyToValueTransform(IHostEnvironment env, IDataView input, string name, string source = null)
```

Comment:
>IHostEnvironment env, IDataView input, string name, string source = null [](start = 35, length = 72)

half of your files formatted in this way, half is one parameter for each line, why?

Reply:
If the line is going to be long enough to fit into the view, I format every parameter on separate line otherwise not. 

For this particular, its fine to have it on one line.